src/include/sof/sof.h

```diff
@@ -104,4 +104,15 @@ struct sof {
 
 struct sof *sof_get(void);
 
+/**
+ * \brief Core-specific firmware context.
+ */
+struct sof_core_context {
+	/* temporary field to inflate the struct, to be removed afterwards */
+	uint32_t dummy;
```

Comment:
Still needed ?

Reply:
Still?

Reply:
The dummy variable is being removed in the next commit in this PR if that's what you mean